tests/unit/tst-completion.test.ts

```diff
@@ -7,7 +7,7 @@ import {
   getCompletionPositionForLine,
   generateCompletionData,
   storeNewDocument,
-} from "./utils";
+} from "./utils.js";
```

Comment:
why are we importing from `./utils.js` here?

Reply:
I think this happens when I execute `xo --fix`. Will change.

Reply:
Deleted js from import in 4074ada0b208018d6f3865ef5c780b7e45b9ab72

Reply:
If `xo --fix` is doing that, can we work out how to turn it off in the .xoconfig?